rust/private/repository_utils.bzl

```diff
@@ -477,9 +485,18 @@ def produce_tool_path(tool_name, target_triple, version):
         tool_name: The name of the tool per static.rust-lang.org
         target_triple: The rust-style target triple of the tool
         version: The version of the tool among "nightly", "beta', or an exact version.
-    """
 
-    return "{}-{}-{}".format(tool_name, version, target_triple)
+    Returns:
```

Comment:
Oh, this is embarassing. This function duplicates most of `produce_tool_suburl`. Would you mind updating `produce_tool_suburl` to call `produce_tool_path`?

Reply:
Done.